bundles/core/org.eclipse.smarthome.core.voice/src/main/java/org/eclipse/smarthome/core/voice/internal/VoiceManagerImpl.java

```diff
@@ -139,20 +141,24 @@ public void say(String text, String voiceId, String sinkId) {
             } else if (voiceId.contains(":")) {
                 // it is a fully qualified unique id
                 String[] segments = voiceId.split(":");
-                tts = ttsServices.get(segments[0]);
-                voice = getVoice(tts.getAvailableVoices(), segments[1]);
+                tts = getTTS(segments[0]);
```

Comment:
Why did you change this? This potentially returns you the default tts instead of the one specified in `segments[0]`, so it unnecessarily complicates things here.